litestar/handlers/websocket_handlers.py

```diff
@@ -232,6 +250,23 @@ async def listener_fn(socket: WebSocket, **kwargs: Any) -> None:
 
         return listener_fn
 
+    def _init_handler_dtos(self) -> None:
+        """Initialize the data and return DTOs for the handler."""
+        data_parameter = self._listener_callback_signature.parameters.get("data")
+        if data_parameter:
```

Comment:
The `data` parameter is required, so this should either use `parameters["data"]`, or throw an exception. Since we validate that a `data` parameter exists, I'd say we don't need to check for it again here or handle the case where it's not present.